test/xpu_api/utilities/function.objects/arithmetic.operations/plus_arifm_op.pass.cpp

```diff
@@ -0,0 +1,86 @@
+// -*- C++ -*-
```

Comment:
Typo in the name of the file:
`plus_arifm_op.pass.cpp` -> `plus_arithm_op.pass.cpp`

Reply:
Fixed